quickstart/101-azure-api-management-create/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
```

Comment:
Why are we only requiring 0.12 for this example. That is super old. Should update to version 1.0 at least.